src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,6 +374,18 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
+        public override bool CursorGrabbed
```

Comment:
What is purpose of Linux native window(work without X11 layer)? Frassle say that if so then there are no window management at all and it is fine to say that we have one window(our window) that has permanent grab(from first review messages branch).